complete/src/main/java/com/example/restservice/service/smsService/SmsManager.java

```diff
@@ -1,18 +1,13 @@
 package com.example.restservice.service.smsService;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 
 @Component
+@RequiredArgsConstructor
 public class SmsManager {
-  private static final Logger LOGGER = LoggerFactory.getLogger(SmsManager.class);
   private final SmsService smsService;
```

Comment:
We’re getting rid of something? Is this constructor no longer necessary?

Reply:
Lombok adds the constructor